fuzzy/docker/base/Dockerfile.ubuntu18.04-verificarlo

```diff
@@ -0,0 +1,43 @@
+FROM ubuntu:18.04
+
+# This Dockerfile is adapted from https://github.com/verificarlo/verificarlo
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update &&\
+    apt install -y autoconf\
+                   automake\
+                   bash ca-certificates\
+                   binutils\
+                   clang-4.0\
+                   gcc-7\
+                   g++-7\
+                   git\
+                   libedit-dev\
+                   libmpfr-dev\
+                   libtool\
+                   libz-dev\
+                   llvm-4.0\
+                   llvm-4.0-dev\
+                   make\
+                   python3\
+                   python3-numpy\
+                   python3-matplotlib
+# test_unstable_branches fails with older versions of llvm
+
+ENV LD_LIBRARY_PATH /usr/lib/gcc/x86_64-linux-gnu/7:/usr/lib/llvm-4.0/lib:/usr/local/lib:$LD_LIBRARY_PATH
+ENV PYTHONPATH /usr/local/lib/python3.6/site-packages/:$PYTHONPATH
+
+# Tests fail with gcc-4.8 (missing symbol in VFC backend)
+# Pytorch requires gcc > 7
+# configure without fortran support as llvm > 3.6
+# Removing setarch from vfc_ddebug as it crashes when Docker is run withouth privileged mode
+RUN git clone https://github.com/verificarlo/verificarlo.git &&\
+    cd verificarlo &&\ 
+    sed -i s,setarch\ \$\(uname\ -m\)\ \-R,,g src/vfc_ddebug &&\
+    ./autogen.sh &&\
+    ./configure --with-llvm=/usr/lib/llvm-4.0/bin/ CC=gcc-7 CXX=g++-7 --without-dragonegg &&\
```

Comment:
Do we know the impact of not using dragonegg? Maybe @pablooliveira could weigh in. If we're happy not using it, I could also finish up the Alpine version.

Reply:
as far as I'm aware it means that we won't be able to compile Fortran.

Reply:
Indeed @glatard; not having `dragonegg` means we cannot compiler fortran. The fuzzy-python image requires `fortran` to compile the `blas` library (used by `numpy`).

Reply:
@ychatelain is working on a verificarlo's PR that will use flang instead of dragonegg to compile fortran. I think it will be released late december or january. This will allow to drop dragonegg.

Reply:
Ok. @glatard would you like to leave this open until then, or install dragonegg in the mean time?